Bottom: efa1ae07b8dcbf227d2be9e117188162ace7a972
Top:    8a15a5512b1c6b56e38558743bc10faccd004cd5
Author: scott snyder <sss@karma>
Date:   2019-12-26 23:57:43 -0500

fix for devil's panties



---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/TheDevilsPanties.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/TheDevilsPanties.java
index c4b9341..f7d98b6 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/TheDevilsPanties.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/TheDevilsPanties.java
@@ -74,7 +74,7 @@ public class TheDevilsPanties extends RandomIndexedComic {
 
 	@Override
 	public String getStripUrlFromId(int num) {
-		return "http://thedevilspanties.com/archives/" + num;
+		return getComicWebPageUrl() + "archives/" + num;
 	}
 
 	@Override
@@ -84,12 +84,12 @@ public class TheDevilsPanties extends RandomIndexedComic {
 
 	@Override
 	protected String getFrontPageUrl() {
-		return "http://thedevilspanties.com/";
+                return getComicWebPageUrl();
 	}
 
 	@Override
 	public String getComicWebPageUrl() {
-		return "http://thedevilspanties.com/";
+		return "https://thedevilspanties.com/";
 	}
 
 	@Override
@@ -125,7 +125,7 @@ public class TheDevilsPanties extends RandomIndexedComic {
 		// set the next and previous comic IDs from the current url's html data
 		setNextId(parseForNextId(final_next, -1));
 		setPreviousId(parseForPrevId(final_prev, -1));
-		return final_str; 
+		return getComicWebPageUrl() + final_str; 
 	}
 
 	@Override
@@ -140,7 +140,7 @@ public class TheDevilsPanties extends RandomIndexedComic {
 		}
 		line = line.replaceAll(".*href=\"", "");
 		line = line.replaceAll("\".*", "");
-		line = line.replaceAll("/archives/", "");
+		line = line.replaceAll(".*/archives/", "");
 		try {
 			return Integer.parseInt(line);
 		}
